### PR TITLE
don't specify write permission where it's irrelevant

### PR DESCRIPTION
**Pull request by fewf**
this addresses https://github.com/denoland/deno/pull/1689#discussion_r254574673

### CODE REVIEW

File: js/read_dir_test.ts [hunk 1]
```diff
@@ -22,12 +22,12 @@ function assertSameContent(files: FileInfo[]) {
...
 }
 
-testPerm({ write: true, read: true }, function readDirSyncSuccess() {
+testPerm({ read: true }, function readDirSyncSuccess() {
   const files = deno.readDirSync("tests/");
   assertSameContent(files);
 });
 
-testPerm({ write: true, read: false }, function readDirSyncPerm() {
+testPerm({ read: false }, function readDirSyncPerm() {
```
Comment (dsseng) [inline js/read_dir_test.ts:30]: Why is this needed?

File: js/read_dir_test.ts [hunk 2]
```diff
@@ -67,12 +67,12 @@ testPerm({ read: true }, function readDirSyncNotFound() {
...
 });
 
-testPerm({ write: true, read: true }, async function readDirSuccess() {
+testPerm({ read: true }, async function readDirSuccess() {
   const files = await deno.readDir("tests/");
   assertSameContent(files);
 });
 
-testPerm({ write: true, read: false }, async function readDirPerm() {
+testPerm({ read: false }, async function readDirPerm() {
```
Comment (dsseng) [inline js/read_dir_test.ts:75]: Maybe just use `test`?

File: js/read_dir_test.ts (hunk 1 above)
Comment (fewf) [reply js/read_dir_test.ts:30]: that's a good point, it's not necessary. the only justification I can think of is that it declares to the next person reading the code which permission is lacking.

**Review by dsseng — APPROVED** [review]
Good! Thanks!
